source/photo.html

```diff
@@ -63,22 +91,21 @@ <h2 class="photo-card__name">Неродные просторы</h2>
                 <p class="photo-card__author">Автор фото: Борис</p>
               </div>
               <div class="post">
-                <button type="button" class="post__like"><svg class="post__like-svg" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="22px" height="20px" viewBox="0 0 22 20" enable-background="new 0 0 22 20" xml:space="preserve">
-                  <g id="Page-1">
-                    <g id="Core" transform="translate(-295.000000, -464.000000)">
-                      <g id="thumb-up" transform="translate(295.000000, 464.000000)">
-                        <path id="Shape" fill="#000000" d="M12.383,3.646l-0.637,2.929L11.219,9H13.7h6.2v2c0,0.018,0,0.037,0.001,0.058L17,18H8V8.029 l0.014-0.015L12.383,3.646 M2,10v8V10 M13.2,0L6.6,6.6C6.2,6.9,6,7.4,6,8v10c0,1.1,0.9,2,2,2h9c0.8,0,1.5-0.5,1.8-1.2l3-7.1 C21.9,11.5,21.9,11.2,21.9,11V9H22c0-1.1-0.9-2-2-2h-6.3l1-4.6V2.1c0-0.4-0.2-0.8-0.4-1.1L13.2,0L13.2,0z M4,8H0v12h4V8L4,8z M22,9c0,0.033,0,0.044,0,0.044V9L22,9z"/>
-                      </g>
-                    </g>
-                  </g>
-                </svg></button>
+                <button type="button" class="post__like">
+                  <svg class="post__like-svg">
+                    <use Xlink:href="#like-svg"></use>
```

Comment:
```suggestion
                    <use xlink:href="#like-svg"></use>
```
(Поправь и у других изображений)